docs/source/changelog.md

```diff
@@ -4,9 +4,49 @@
+[event loop](https://docs.python.org/3/library/asyncio-eventloop.html).
+
+A key motivation for this change was that it reduced the complexity of the
+project's code, but it has also been found to reduce transient issues within the
+k8s cluster. For more technical details about this, see
+[#563](https://github.com/jupyterhub/kubespawner/pull/563).
+
+#### Breaking changes
+
+- Support for Python 3.6 dropped
+- The configuration `k8s_api_threadpool_workers` is deprecated as we don't
+  create threads any more, but now instead relies on scheduling everything to
+  run in an event loop.
+- A dependency on the library
+  [`kubernetes`](https://github.com/kubernetes-client/python#readme) is replaced
+  with a dependency on
+  [`kubernetes_asyncio`](https://github.com/tomplus/kubernetes_asyncio#readme).
+
+#### Bugs fixed
+
+- close shared clients when loop closes [#579](https://github.com/jupyterhub/kubespawner/pull/579) ([@minrk](https://github.com/minrk), [@consideRatio](https://github.com/consideRatio))
+
+#### Maintenance and upkeep improvements
+
+- Simplify async init [#576](https://github.com/jupyterhub/kubespawner/pull/576) ([@minrk](https://github.com/minrk), [@consideRatio](https://github.com/consideRatio))
+- Rely on the event loop: use `kubernetes_asyncio` instead of `kubernetes` and dedicated threads [#563](https://github.com/jupyterhub/kubespawner/pull/563) ([@athornton](https://github.com/athornton), [@yuvipanda](https://github.com/yuvipanda), [@consideRatio](https://github.com/consideRatio), [@minrk](https://github.com/minrk), [@manics](https://github.com/manics))
+
+#### Contributors to this release
+
+([GitHub contributors page for this release](https://github.com/jupyterhub/kubespawner/graphs/contributors?from=2022-02-15&to=2022-03-10&type=c))
+
+[@athornton](https://github.com/search?q=repo%3Ajupyterhub%2Fkubespawner+involves%3Aathornton+updated%3A2022-02-15..2022-03-10&type=Issues) | [@choldgraf](https://github.com/search?q=repo%3Ajupyterhub%2Fkubespawner+involves%3Acholdgraf+updated%3A2022-02-15..2022-03-10&type=Issues) | [@consideRatio](https://github.com/search?q=repo%3Ajupyterhub%2Fkubespawner+involves%3AconsideRatio+updated%3A2022-02-15..2022-03-10&type=Issues) | [@manics](https://github.com/search?q=repo%3Ajupyterhub%2Fkubespawner+involves%3Amanics+updated%3A2022-02-15..2022-03-10&type=Issues) | [@minrk](https://github.com/search?q=repo%3Ajupyterhub%2Fkubespawner+involves%3Aminrk+updated%3A2022-02-15..2022-03-10&type=Issues) | [@rccern](https://github.com/search?q=repo%3Ajupyterhub%2Fkubespawner+involves%3Arccern+updated%3A2022-02-15..2022-03-10&type=Issues) | [@welcome](https://github.com/search?q=repo%3Ajupyterhub%2Fkubespawner+involves%3Awelcome+updated%3A2022-02-15..2022-03-10&type=Issues) | [@yuvipanda](https://github.com/search?q=repo%3Ajupyterhub%2Fkubespawner+involves%3Ayuvipanda+updated%3A2022-02-15..2022-03-10&type=Issues)
+
 ## 2.0
 
-### 2.0.1 - 2022-02-15
+### [2.0.1] - 2022-02-15
```

Comment:
Is the addition of `[ ... ]` a workaround for https://github.com/executablebooks/MyST-Parser/issues/527 ?

Reply:
@manics no it is just a practice that I've kept alive that was introduced a very long time ago in some JupyterHub project.